x/ibc-account/keeper/relay.go

```diff
@@ -40,7 +40,13 @@ func (k Keeper) TryRegisterIBCAccount(ctx sdk.Context, sourcePort, sourceChannel
 		Data: salt,
 	}
 
-	// TODO: Add timeout height and timestamp
+	timeoutHeight := clienttypes.Height{
+		RevisionNumber: 2,
+		RevisionHeight: ^uint64(0),
+	}
+
+	const timeoutTimestamp = ^uint64(0)
```

Comment:
might be good to add a comment on why we set the timeout to be as large as possible

Reply:
> Another feedback I had was to change the `module name` to `github.com/cosmos/interchain-account` since the repo is moved. This requires to update all proto files and imports inside both modules. It can be handled with another PR.

Agreed. Lets tackle this in a separate PR.